diff --git a/node_modules/snarkjs/build/main.cjs b/node_modules/snarkjs/build/main.cjs
index 75dfe1c..76bb47b 100644
--- a/node_modules/snarkjs/build/main.cjs
+++ b/node_modules/snarkjs/build/main.cjs
@@ -5189,12 +5189,12 @@ async function phase2verifyFromInit(initFileName, pTauFileName, zkeyFileName, lo
             if (logger) logger.info(`Beacon iterations Exp: ${c.numIterationsExp}`);
         }
     }
+
     if (logger) logger.info("-------------------------");
 
     if (logger) logger.info("ZKey Ok!");
 
-    return true;
-
+    return mpcParams;
 
     async function sectionHasSameRatio(groupName, fd1, sections1, fd2, sections2, idSection, g2sp, g2spx, sectionName) {
         const MAX_CHUNK_SIZE = 1<<20;
